fhir-server-test/src/test/java/org/linuxforhealth/fhir/server/test/BundleTest.java

```diff
@@ -127,6 +155,14 @@ public void retrieveConfig() throws Exception {
 
         deleteSupported = isDeleteSupported();
         System.out.println("Delete operation supported?: " + deleteSupported.toString());
+        
+        kafkaAuditEnabled = isAuditLogSupported();
+        logger.info("kafkaAuditEnabled flag " +kafkaAuditEnabled);
```

Comment:
very minor, but we usually use spaces before and after the `+` operator
```suggestion
        logger.info("kafkaAuditEnabled flag " + kafkaAuditEnabled);
```
applies to a number of places in this PR

Reply:
This has been fixed as suggested.